sdk/storage/Azure.Storage.Common/src/Azure.Storage.Common.csproj

```diff
@@ -20,7 +20,8 @@
     <IncludeOperationsSharedSource>true</IncludeOperationsSharedSource>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Core" />
+    <!-- <PackageReference Include="Azure.Core" /> -->
+    <ProjectReference Include="..\..\..\core\Azure.Core\src\Azure.Core.csproj" />
```

Comment:
Not sure that I understand why this is needed.

Reply:
But there's no explicit use of the new features/types here.   That's why I'm asking.  😄

Reply:
Same comment as above.